### ISSUE TITLE
Add helpers to generate signatures for AENS transactions in contract

### ISSUE THREAD

**Issue by noandrea** [aeternity/aepp-sdk-python#306]
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

* **What is the expected behavior?**

Helper functions for example like 
`Aens.transfer_signature(owner: Account, name, contract_address)`

see this https://github.com/aeternity/protocol/blob/master/contracts/sophia.md#aens-interface

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**
  
  - Node Version: v0.0.0
  - Protocol Version: 1
  - Compiler version: v0.0.0
  - VM Version: aevm | fate
  - SDK Version: v0.0.0
  - Python version: v3.7.0

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. forum, telegram, etc)

**Comment by noandrea** [comment]
the feature is implemented in #353 
but due to apparent inconsistencies with the documentation [here](https://github.com/aeternity/protocol/blob/aeternity-node-v5.4.1/contracts/sophia.md#aens-interface) and the implementation 
tests are disabled since this error is reported by the node for the AENS.claim

```
DEBUG    aeternity.openapi:openapi.py:210 POST http://localhost:3080/compile, params:{}, body: {'code': 'contract DelegateTest =\n  // Transactions\n  stateful payable entrypoint signedPreclaim(addr  : address,\n                                             chash : hash,\n                                             sign  : signature) : uint =\n    AENS.preclaim(addr, chash, signature = sign)\n\n  stateful entrypoint signedClaim(addr : address,\n                                name : string,\n                                salt : int,\n                                sign : signature) : unit =\n    AENS.claim(addr, name, salt, signature = sign)\n\n\n  stateful entrypoint signedTransfer(owner     : address,\n                                   new_owner : address,\n                                   name      : string,\n                                   sign      : signature) : unit =\n    AENS.transfer(owner, new_owner, name, signature = sign)\n\n  stateful entrypoint signedRevoke(owner     : address,\n                                   name      : string,\n                                   sign      : signature) : unit =\n    AENS.revoke(owner, name, signature = sign)\n', 'options': {'backend': 'fate'}} --> [{"message":"Unbound type uint (at line 5, column 67).\n","pos":{"col":67,"line":5},"type":"type_error"},{"context":"when checking the type of the expression at line 6, column 5\n  AENS.preclaim(addr, chash, signature = sign) : unit\nagainst the expected type\n  uint\n","message":"Cannot unify unit\n         and uint\n","pos":{"col":5,"line":6},"type":"type_error"},{"context":"when checking the application at line 12, column 5 of\n  AENS.claim :\n    (signature : signature, address, string, int, int) => unit\nto arguments\n  addr : address\n  name : string\n  salt : int\n","message":"Cannot unify int\n         and \n","pos":{"col":5,"line":12},"type":"type_error"}]

```

**Comment by noandrea** [comment]
closed by #262
